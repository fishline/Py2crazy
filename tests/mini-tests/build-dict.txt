w = {"joe" : 5, "mindy" : 6, "jack" : 7}
