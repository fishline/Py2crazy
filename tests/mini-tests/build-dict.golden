Disassembling top-level module in ./mini-tests/build-dict.txt
  1       0 BUILD_MAP            w = [43m[30m{"joe" : 5, "mindy" : 6, "jack" : 7}[39m[49m
          3 LOAD_CONST           w = {"joe" : [43m[30m5[39m[49m, "mindy" : 6, "jack" : 7}
          6 LOAD_CONST           w = {[43m[30m"joe"[39m[49m : 5, "mindy" : 6, "jack" : 7}
          9 STORE_MAP            w = [43m[30m{"joe" : 5, "mindy" : 6, "jack" : 7}[39m[49m
         10 LOAD_CONST           w = {"joe" : 5, "mindy" : [43m[30m6[39m[49m, "jack" : 7}
         13 LOAD_CONST           w = {"joe" : 5, [43m[30m"mindy"[39m[49m : 6, "jack" : 7}
         16 STORE_MAP            w = [43m[30m{"joe" : 5, "mindy" : 6, "jack" : 7}[39m[49m
         17 LOAD_CONST           w = {"joe" : 5, "mindy" : 6, "jack" : [43m[30m7[39m[49m}
         20 LOAD_CONST           w = {"joe" : 5, "mindy" : 6, [43m[30m"jack"[39m[49m : 7}
         23 STORE_MAP            w = [43m[30m{"joe" : 5, "mindy" : 6, "jack" : 7}[39m[49m
         24 STORE_NAME           [43m[30mw[39m[49m = {"joe" : 5, "mindy" : 6, "jack" : 7}
         27 LOAD_CONST           [43m[30mw[39m[49m = {"joe" : 5, "mindy" : 6, "jack" : 7}
         30 RETURN_VALUE         [43m[30mw[39m[49m = {"joe" : 5, "mindy" : 6, "jack" : 7}
